bin/bot.js

```diff
@@ -73,7 +73,8 @@ client.on('message', message => {
 	if (!perms) return;
 
 	const userId = message.author.id;
-	const id = message.id + ', ' + userId;
+	const userName = message.author.username;
+	const id = 'msgId: ' + message.id + ', userId: ' + userId + ', userName: ' + userName;
```

Comment:
I guess it depends on how you want to use the log file. If you want to throw it in Splunk or some other search/analysis engine, then just having the ID is fine, but I'd argue that you can't really do that effectively w/o the timestamp (added below). Also, if you just want to see, "what did this person do?" then just the ID is fine. However, if you want to see, "who did what and when?" then having the user name, even if it changes occasionally or has some non-printable characters, is essential.

Notice I didn't remove the ID; I just added the user name.

Regarding whether the user name is allowed to be logged, you're already logging the user ID, which is personally identifiable information (PII), so the user name is no worse. 

Having said all this, if you really object to the user name, I'll remove it.

Reply:
I do object. I do not feel it adds any transparency.

More important would be logging guild and channel information. However, I do not want that either because two more id values would effectively increase the byte growth of the log file by about 30%. Additionally, you can look up this values with the back end REPL.